src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/serverprocess.h

```diff
@@ -6,7 +6,13 @@
 #include <random>
 #include <map>
 
+// Keep using 1025 to avoid breaking existing services that are choosing
+// a port lower than 10000 via the ASPNETCORE_PORT environment variable
 #define MIN_PORT                                    1025
+// This value is somewhat arbitrary. Fixed-port services tend to
```

Comment:
Improved comment based on new distinction.

```suggestion
// Minimum number used for automatic random port assignment.
// This value is somewhat arbitrary. Fixed-port services tend to
```